pywinauto/unittests/test_mouse.py

```diff
@@ -128,6 +129,33 @@ def test_wheel_click(self):
         self.assertTrue("Mouse Release" in data)
         self.assertTrue("MiddleButton" in data)
 
+    if sys.platform == "win32":
+        def test_mouse_can_move_cursor1(self):
```

Comment:
Please add `# TODO: make the feature and the tests cross-platform (duration param)`

Reply:
added, 18434a4686040bcad4522a37e7e1bc6932370975